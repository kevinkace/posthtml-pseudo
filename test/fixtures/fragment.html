<div>
    <h1>header</h1>
    <p>para</p>
</div>