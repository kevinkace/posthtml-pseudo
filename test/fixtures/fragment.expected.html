<div>
    <h1 class=":first-child :first-of-type :last-of-type :only-of-type">header</h1>
    <p class=":first-of-type :last-child :last-of-type :only-of-type">para</p>
</div>